test/test_checks.py

```diff
@@ -1,6 +1,18 @@
 from flake8_quotes import QuoteChecker
 import os
-from unittest import TestCase
+import subprocess
+from unittest import expectedFailure, TestCase
+
+
+FLAKE8_VERSION = int(os.environ.get('FLAKE8_VERSION', 2))
```

Comment:
Oh and a string only hides any issues. Because if it's actually using a proper version number the check below will **silently** fail, unless we do some version number parsing or are careful what version we actually use.